formiodata/components.py

```diff
@@ -17,6 +17,11 @@ def __init__(self, raw, builder, **kwargs):
         # i18n (language, translations)
         self.language = kwargs.get('language', 'en')
         self.i18n = kwargs.get('i18n', {})
+        self.resources = kwargs.get('resources', {})
+        if self.resources and isinstance(self.resources, str):
+            self.resources = json.loads(self.resources)
+        self.html = ""
```

Comment:
Why the `html` property (also used in the new `render` function) ? 
I notice the `html` property is most of times not present in the Builder components.
Please explain the aim.

Reply:
I'm working on a spinoff of this library, which goal is server side rendering, with jinja2.
If anyone have my same need overriding render method is possible to pass template and engine, to rendering your custom theme with minimal js libs.
In our  use case we use Formio  only as a builder facility in development process, but not in font-end side in applications services.

Reply:
Thanks for clarifying!